src/components/common/DatePicker.js

```diff
@@ -18,24 +18,22 @@ const useStyles = makeStyles(theme => ({
 
 const DatePicker = props => {
   const classes = useStyles();
-  const proceduresList = props.procedures.proceduresList;
-  const targetProcedure = (proceduresList.find(item => item.id === props.id) || {});
-  const targetDate = (targetProcedure)
-
-  console.log(targetProcedure)
-
+  const dateNow = new Date();
+  const yearNow = dateNow.getFullYear();
+  const monthNow = dateNow.getMonth();
+  const dayNow = dateNow.getDate();
 
   function editDate(event) {
       props.editProcedureDate(event.target.value);
   }
 
```

Comment:
change name and use es6
For instance

const handleDate = event => props.editProcedureDate(event.target.value);